sounds/views.py

```diff
@@ -253,12 +254,33 @@ def sound(request, username, sound_id):
         'display_random_link': display_random_link,
         'do_log': do_log,
         'is_following': is_following,
-        'is_explicit': is_explicit
+        'is_explicit': is_explicit,
     }
     tvars.update(paginate(request, qs, settings.SOUND_COMMENTS_PER_PAGE))
     return render(request, 'sounds/sound.html', tvars)
 
 
+@login_required
+def after_download_modal(request):
+    """
+    This view checks if a modal should be shown after the user has downloaded a sound, and returns either the contents
+    of the modal if needed.
+    """
+    response = HttpResponse()  # Default empty response with no content
+    sound_name = request.GET.get('sound_name', 'this sound')  # Gets some data sent by the client
+
+    if settings.AFTER_DOWNLOAD_MODAL == settings.AFTER_DOWNLOAD_MODAL_DONATION:
+        times_shown_in_last_day = int(request.GET.get('num_times_shown', 0))
```

Comment:
Fixed in https://github.com/MTG/freesound/pull/820/commits/b70d4f2407a427f288da5a041c72cbef7d5cddcb.